eidas_node/cznic/templates/eidas_node/connector/base.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n static %}
```

Comment:
I think we should create a `{% block content %}` and `{% block footer %}` in the main template and extend it here. That would allow it to be more general and flexible.

Reply:
Done.